app/views/lists/index.html.erb

```diff
@@ -1,4 +1,4 @@
-<h1> Your shopping Lists</h1>
+<h1> Your shopping Lists </h1>
```

Comment:
h1 should be changed to h2
There should only be one h1 on the app, on the homepage, for SEO purposes